tsconfig.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Could we use this config: https://github.com/Rokt33r/hast-util-to-html/blob/318112f2c59275c59d8b687f75f4c21b12486f44/types/tsconfig.json ?

Reply:
will just need to add `"noemit": true` because i have type tests that dont compile to js

Reply:
Using dtslint to do the checking should remove the need for `"noemit": true`

Reply:
that's weird but ok haha